src/ui/FileViewer/index.tsx

```diff
@@ -138,23 +108,50 @@ export const FileViewerComponent = ({
           )
         }
       </div>
+      <ImagesView props={props} stringSet={stringSet}/>
     </div>
   );
 };
 
 export interface FileViewerProps {
   message: FileMessage;
+  onClose: (e: MouseEvent) => void;
   isByMe?: boolean;
-  onClose: (e: MouseEvent<HTMLDivElement>) => void;
-  onDelete: (e: MouseEvent<HTMLDivElement>) => void;
+  onDelete?: (e: MouseEvent) => void;
+  currentIndex?: number;
+  onClickLeft?: () => void;
+  onClickRight?: () => void;
 }
 
 export default function FileViewer({
   message,
-  isByMe = false,
   onClose,
+  isByMe = false,
   onDelete,
+  currentIndex,
+  onClickLeft,
+  onClickRight,
 }: FileViewerProps): ReactElement {
+  if (message?.isMultipleFilesMessage()) {
```

Comment:
Maybe `message?.isMultipleFilesMessage?.() `

Reply:
no they are all required
```message.isMultipleFilesMessage() ```
is correct